### PR TITLE
CSS module project: multi-column layout

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: files/en-us/web/css/css_multicol_layout/index.md [hunk 1]
```diff
@@ -6,41 +6,37 @@ spec-urls: https://drafts.csswg.org/css-multicol/
...
+  - {{cssxref("column-width")}}
 
-### Properties related to CSS fragmentation
-
-- {{cssxref("break-after")}}
-- {{cssxref("break-before")}}
-- {{cssxref("break-inside")}}
-- {{cssxref("orphans")}}
-- {{cssxref("widows")}}
+> **Note:** Setting container height and line length can pose challenges for people with visual or cognitive disabilities. WCAG Success Criterion state that, even when the text size is doubled, content should not need to be scrolled.
```
Comment (dipikabh) [inline files/en-us/web/css/css_multicol_layout/index.md:39]:
```suggestion
> **Note:** Bear in mind that setting container height and line length can pose challenges for people with visual or cognitive disabilities. WCAG Success Criterion states that even when the text size is doubled, content should not need to be scrolled.
```
could we add a link to the "WCAG Success Criterion"?

File: files/en-us/web/css/css_multicol_layout/index.md [hunk 2]
```diff
@@ -49,19 +45,31 @@ Multi-column layout is closely related to [paged media](/en-US/docs/Web/CSS/CSS_
...
 
-Other CSS layout technologies include:
-
-- [CSS flexible box layout](/en-US/docs/Web/CSS/CSS_flexible_box_layout) (CSS flexbox)
-- [CSS grid layout](/en-US/docs/Web/CSS/CSS_grid_layout)
+- [Learn: multiple-column layout](/en-US/docs/Learn/CSS/CSS_layout/Multiple-column_Layout)
+- [CSS paged media](/en-US/docs/Web/CSS/CSS_paged_media) module
+- [CSS fragmentation](/en-US/docs/Web/CSS/CSS_fragmentation) module
+- [CSS flexible box layout](/en-US/docs/Web/CSS/CSS_flexible_box_layout) module
+- [CSS grid layout](/en-US/docs/Web/CSS/CSS_grid_layout) module
```
Comment (dipikabh) [inline files/en-us/web/css/css_multicol_layout/index.md:75]:
```suggestion
- [CSS fragmentation](/en-US/docs/Web/CSS/CSS_fragmentation) module
- [CSS flexible box layout](/en-US/docs/Web/CSS/CSS_flexible_box_layout) module
- [CSS grid layout](/en-US/docs/Web/CSS/CSS_grid_layout) module
- [CSS paged media](/en-US/docs/Web/CSS/CSS_paged_media) module
```
moved 'paged media' to the end

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks @estelle. I've added a few edit suggestions.

File: files/en-us/web/css/css_multicol_layout/index.md [hunk 3]
```diff
@@ -6,41 +6,37 @@ spec-urls: https://drafts.csswg.org/css-multicol/
...
+{{EmbedGHLiveSample("css-examples/modules/multicol.html", '100%', 650)}}
 
-{{EmbedGHLiveSample("css-examples/multicol/basics/column-count.html", '100%', 550)}}
+To see the code for this columned layout, [view the source on GitHub](https://github.com/mdn/css-examples/blob/main/modules/multicol.html).
 
-## Relationship to fragmentation
-
-Multi-column layout is closely related to [paged media](/en-US/docs/Web/CSS/CSS_paged_media), in that each column box becomes a fragment, much like a printed page becomes a fragment of an overall document. Therefore, the properties now defined in the [CSS fragmentation](/en-US/docs/Web/CSS/CSS_fragmentation) specification are required to control how content breaks between columns.
+> **Note:** Multiple-column layout is closely related to [paged media](/en-US/docs/Web/CSS/CSS_paged_media), as each column box is a fragment, much like each printed page is a fragment of a document. Properties defined in [CSS fragmentation](/en-US/docs/Web/CSS/CSS_fragmentation) control how content breaks between columns.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/css_multicol_layout/index.md]: Good catch!

**Comment by estelle** [comment]
Thanks for the review. I committed all the edits and addressed all the suggestions.

**Pull request closed without merging**